docs/developers/erc_20_integration.md

```diff
@@ -0,0 +1,82 @@
+<!--
+order: 3
+-->
+
+# ERC-20 Registration
+
+Learn how to register interoperable ERC-20s through [Evmos Governance](../users/governance/overview.md). {synopsis}
+
```

Comment:
add a recap of why anyone would want to register an erc20

Reply:
also add a link to our medium article: https://medium.com/evmos/introducing-evmos-erc20-module-f40a61e05273